tensorflow-framework/src/test/java/org/tensorflow/framework/activations/GeLUTest.java

```diff
@@ -0,0 +1,93 @@
+package org.tensorflow.framework.activations;
+
+import org.junit.jupiter.api.Test;
+import org.tensorflow.Operand;
+import org.tensorflow.framework.utils.TestSession;
+import org.tensorflow.op.Ops;
+import org.tensorflow.types.TFloat32;
+import org.tensorflow.types.TFloat64;
+
+class GeLUTest {
+
+  private final TestSession.Mode[] tfModes = {TestSession.Mode.EAGER, TestSession.Mode.GRAPH};
+
+  /** Test of GeLU call method */
+  @Test
+  public void testCallFloat() {
+    float[][] input = {
+      {0.22805803f, 0.60407318f, 0.91519962f, 0.35643331f, 0.28702669f},
```

Comment:
Can we test some negative values here? Specifically in the region -2,0 where it's negative, and something more negative like -10 where it should be approximately zero.